### ISSUE TITLE
Reorder and relabel in-entity attributes in Detail subwindow

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#361]
The issue relates to this spot in the GUI:
![obrazek](https://user-images.githubusercontent.com/55400794/133890924-a8506c36-433d-4ba2-8ce6-38327f019743.png)

Necessary improvements:
- As previously communicated, follow the order of cols in the Action types list. (Apart from Status - there I agree with its current placement.)
- Follow the attribute names in Action types list rather then reinventing them.
- "Entity Subject" listed three times, while actant1 and actant2 allowed entity type is lacking (error).

In other words, the attribute labels should be as follows, and in the order as listed here:
Label
Detail
Status
Language
Subject entity type
Subject valency
Actant1 entity type
Actant1 valency
Actant2 entity type
Actant2 valency
Notes

Furthermore, put the entity types with first uppercase letter: e.g., "Person" and not "person".
